### DISCUSSION TITLE
How to select a version tag of ASP.NET Core source code

### DISCUSSION THREAD

**Discussion by Rick-Anderson** [dotnet/AspNetCore.Docs#26205] [Q&A] [answered]
Select the **Tags** tab and then select the version of ASP.NET Core.

![image](https://user-images.githubusercontent.com/3605364/174899887-ae23c0f3-13db-4b52-8212-8ce497d650e9.png)

**Comment by dhanur350** [comment]
Select the tags tab and then select your desired version required

**Comment by Rick-Anderson** [comment, marked as answer]
Select the **Tags** tab and then select the version of ASP.NET Core.
